releasenotes/notes/use-bound-pass-manager-in-circuitqnn-20c60effedb70d76.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Fix in ``CircuitQNN`` that ensures unbound_pass_manager is called when caching
+    the QNN circuit and that ``bound_pass_manager`` is called when QNN parameters
+    are assigned.
```

Since this is a new feature, please don't use the word "fix":
```suggestion
    Added a new feature in ``CircuitQNN`` that ensures unbound_pass_manager is called when caching
    the QNN circuit and that ``bound_pass_manager`` is called when QNN parameters
    are assigned.
```